src/components/button/Button.tsx

```diff
@@ -27,9 +32,33 @@ export function Button(props: ButtonProps) {
       fill={tooltipProps?.fill || buttonProps.fill}
       {...tooltipProps}
       renderTarget={({ isOpen, ...targetProps }) => (
-        <InnerButton {...targetProps} {...buttonProps}>
-          {children}
-        </InnerButton>
+        <div style={{ position: 'relative' }}>
+          {tag && (
+            <Tag
+              style={{
+                position: 'absolute',
+                top: 0,
+                left: 0,
+                cursor: 'default',
+                fontSize: '0.875em',
+                padding: 2,
+                minHeight: 15,
+                minWidth: 15,
+                lineHeight: '1em',
+                flexDirection: 'column-reverse',
```

Comment:
Is this really useful?

Reply:
Yes to make it centred

Reply:
`align-text: center;` is better then (mainly because more explicit about why the property is there)

Reply:
I used `flexDirection: 'column'` because the default was `flexDirection: 'row'`, and nothing make it centered

Reply:
It works but isn't it clearer to just use the `align-text` property in order to align (center) the text?

Reply:
Using the dev tools it seems to work as well